src/main.go

```
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+func statusHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprint(w, "I'm well, thank you.")
+}
+
+func metricsHandler(w http.ResponseWriter, r *http.Request) {
+	jmxMetrics, err := ReadJmxMetrics()
+	if err == nil {
```

you're not doing anything here if `ReadJMXMetrics()` returns an error - I imagine you will want to return an error to the client and perhaps log an error.

generally you want to check `if err != nil`, process the error and (usually) return it before anything else